examples/load-test/main.tf

```diff
@@ -0,0 +1,73 @@
+terraform {
+	required_providers {
+		scylladbcloud = {
+			source = "registry.terraform.io/scylladb/scylladbcloud"
+		}
+	}
+}
+
+provider "aws" { }
+
+provider "scylladbcloud" {
+	token = "..."
+}
+
+resource "aws_vpc" "bench" {
+	cidr_block = "10.0.0.0/16"
+}
+
+resource "scylladbcloud_cluster" "mycluster" {
+	name       = "Load Test"
+	byoa_id    = 1002
+	region     = "us-east-1"
+	node_count = 3
+	node_type  = "i3.xlarge"
+	cidr_block = "172.31.0.0/16"
+
+	enable_vpc_peering = true
+	enable_dns         = true
```

Comment:
is `byoa_id` and `enable_dns` needed for this example? I would keep it as simple as possible